src/node_crypto.cc

```diff
@@ -4841,6 +4850,8 @@ void PBKDF2(const FunctionCallbackInfo<Value>& args) {
     Local<Value> argv[2];
     EIO_PBKDF2(req);
     EIO_PBKDF2After(req, argv);
+    req->release();
+    delete req;
```

Comment:
I'd call release() from the destructor because I honestly don't see a good reason for the current approach where you have to call release() manually.

If you change that, make sure that pass_, salt_ and key_ are set to `nullptr` so that multiple release() calls are idempotent.